doc/glossary.rst

```diff
@@ -0,0 +1,1216 @@
+        needed for full verbosity.  Verbose logs are usually printed to
+        Standard Output.
+        Estimators should not produce any output on Standard Output with the
+        default ``verbose`` setting.
+
+    ``warm_start``
+
+        When fitting an estimator repeatedly on the same dataset, but for
+        multiple parameter values (such as to find the value maximizing
+        performance as in :ref:`grid search <grid_search>`), it may be possible
+        to reuse aspects of the model learnt from the previous parameter value,
+        saving time.  When ``warm_start`` is true, the existing :term:`fitted`
+        model :term:`attributes` an are used to initialise the new model
+        in a subsequent call to :term:`fit`.
+
+        Note that this is only applicable for some models and some
+        parameters, and even some orders of parameter values. For example,
+        ``warm_start`` may be used when building random forests to add more
+        trees to the forest (increasing ``n_estimators``) but not to reduce
+        their number.
+
+        :term:`partial_fit` also retains the model between calls, but differs:
+        with ``warm_start`` the parameters change and the data is constant
+        across calls to ``fit``; with ``partial_fit``, the mini-batch of data
+        changes and model parameters stay fixed.
+
+.. _glossary_attributes:
+
+Attributes
+==========
+
+See concept :term:`attribute`.
+
+.. glossary::
+
+    ``classes_``
+        A list of class labels known to the :term:`classifier`, mapping each
+        label to a numerical index used in the model representation our output.
+        For instance, the array output from :term:`predict_proba` has columns
+        aligned with ``classes_``. For :term:`multi-output` classifiers,
+        ``classes_`` should be a list of lists, with one class listing for
+        each output.  For each output, the classes should be sorted
+        (numerically or using string comparison).
+
+        ``classes_`` and the mapping to indices is often managed with
+        :class:`preprocessing.LabelEncoder`.
+
+    ``components_``
+        An affine transformation matrix of shape ``(n_components, n_features)``
+        used in many linear :term:`transformers` where ``n_components`` is the
+        number of output features and ``n_features`` is the number of input
+        features.
+
+        See also :term:`components_` which is a similar attribute for linear
+        predictors.
+
+    ``coef_``
+        The weight/coefficient matrix of a generalised linear model
+        :term:`predictor`, of shape ``(n_features,)`` for binary classification
+        and single-output regression, ``(n_classes, n_features)`` for
+        multiclass classification and ``(n_targets, n_features)`` for
+        multi-output regression. Note this does not include the intercept,
+        or bias, term, which is stored in ``intercept_``.
```

Comment:
I guess the comma after bias is correct? Reads odd to me but you're the native speaker ;)

Reply:
No, it's not good. `Note this does not include the intercept - or bias - term, which is stored in intercept_.`

Reply:
totally up to you.